charts/consul/Chart.yaml

```diff
@@ -18,7 +18,7 @@ annotations:
     - name: consul
       image: docker.mirror.hashicorp.services/hashicorppreview/consul:1.18-dev
     - name: consul-k8s-control-plane
-      image: docker.mirror.hashicorp.services/hashicorppreview/consul-k8s-control-plane:1.4-dev
+      image: docker.mirror.hashicorp.services/hashicorppreview/consul-k8s-control-plane:1.4.2-dev
```

Comment:
I think we want to keep this at the floating tag similar to [`main`](https://github.com/hashicorp/consul-k8s/blob/main/charts/consul/Chart.yaml)